src/modules/sensors/voted_sensors_update.cpp

```diff
@@ -405,32 +405,48 @@ void VotedSensorsUpdate::setRelativeTimestamps(sensor_combined_s &raw)
 
 void VotedSensorsUpdate::calcAccelInconsistency()
 {
-	const Vector3f primary_accel{_last_sensor_data[_accel.last_best_vote].accelerometer_m_s2};
-
-	// Check each sensor against the primary
+	Vector3f accel_mean = {};
+	Vector3f accel_all[GYRO_COUNT_MAX];
+	uint8_t accel_count = 0;
 	for (int sensor_index = 0; sensor_index < ACCEL_COUNT_MAX; sensor_index++) {
-		// check that the sensor we are checking against is not the same as the primary
-		if (_accel.advertised[sensor_index] && (_accel.priority[sensor_index] > 0)
-		    && (_accel_device_id[sensor_index] != _selection.accel_device_id)) {
-
-			const Vector3f current_accel{_last_sensor_data[sensor_index].accelerometer_m_s2};
-			_accel_diff[sensor_index] = 0.95f * _accel_diff[sensor_index] + 0.05f * (primary_accel - current_accel);
+		if (_accel_device_id[sensor_index] != 0) {
```

Comment:
```suggestion
		if ((_accel_device_id[sensor_index] != 0) && (_accel.priority[sensor_index] > 0)) {
```

Priority 0 means sensor is disabled.

Reply:
Done.